app/js/SpellingAPIController.js

```diff
@@ -49,6 +49,19 @@ module.exports = {
     })
   },
 
+  unlearn(req, res, next) {
+    metrics.inc('spelling-unlearn', 0.1)
+    const { token, word } = extractLearnRequestData(req)
+    logger.info({ token, word }, 'unlearning word')
+    SpellingAPIManager.unlearnWord(token, req.body, function(error) {
+      if (error != null) {
+        return next(error)
+      }
+      res.sendStatus(200)
```

Comment:
```suggestion
      res.sendStatus(204)
```

No Content fits better IMHO.

Reply:
Yeah, I think you're right. I considered that but decided to stick with consistency WRT the `learn` endpoint. I'd need to check if there are any clients that will be explicitly listening for 200.

Reply:
+1 for consistency between the 2 endpoints, I'm happy with either change to `204` or keep `200`.